src/types/module-deployments/DropModuleMetadata.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  JsonObject,
+  JsonProperty,
+  PropertyConvertingMode,
+} from "json2typescript";
+import CommonModuleMetadata from "./CommonModuleMetadata";
+
+@JsonObject("DropModuleMetadata")
+export default class DropModuleMetadata extends CommonModuleMetadata {
+  /**
+   * The amount of royalty collected on all royalties represented as basis points.
+   * The default is 0 (no royalties).
+   *
+   * 1 basis point = 0.01%
+   *
+   * For example: if this value is 100, then the royalty is 1% of the total sales.
+   */
+  @JsonProperty(
+    "seller_fee_basis_points",
+    Number,
+    PropertyConvertingMode.IGNORE_NULLABLE,
+  )
+  sellerFeeBasisPoints = 0;
+
+  /**
+   * The symbol for the Drop Collection
+   */
+  @JsonProperty("symbol", String, PropertyConvertingMode.IGNORE_NULLABLE)
+  symbol?: string;
+
+  /**
+   * The max supply (required)
+   */
+  @JsonProperty("max_supply", Number, PropertyConvertingMode.IGNORE_NULLABLE)
+  maxSupply = 1;
```

Comment:
I think we probably want to default to 0 or if it's required, we probably don't need a default value because its kinda confusing

Reply:
It's just to get the type checking, it doesn't actually do anything. I need a value though so its initialized to something.

Reply:
in general i think the type should really be a BigNumberish, but i'm not sure how that works with json2ts.
i think if that's the case we should default it to 0, i think `1` feels arbitrary.

Reply:
I can write a converter for json2ts, its simple and def something we want for the rest of the SDK.